src/java/org/apache/cassandra/guardrails/Guardrail.java

```diff
@@ -59,64 +59,55 @@
 {
     private static final NoSpamLogger logger = NoSpamLogger.getLogger(LoggerFactory.getLogger(Guardrail.class),
                                                                       10, TimeUnit.MINUTES);
-
     private static final String REDACTED = "<redacted>";
 
+    /** A name identifying the guardrail (mainly for shipping with Insights events). */
```

Comment:
nit: for clarity, I suggest to keep fields in yaml, Config, GuardrailsConfig and Guardrails in the same order (same as in yaml)

Reply:
I tend to agree, but for now I think keeping these consistent with current cndb code is of greater benefit to make the Astra transition easier.